docker/Dockerfile-opflex-build-base

```diff
@@ -5,13 +5,30 @@ RUN microdnf install --enablerepo codeready-builder-for-rhel-8-x86_64-rpms \
     libtool pkgconfig autoconf automake make cmake file python2-six \
     openssl-devel git gcc gcc-c++ boost-devel diffutils python2-devel \
     libnetfilter_conntrack-devel wget which curl-devel procps zlib-devel \
+    libmnl-devel vi \
     && microdnf clean all
+RUN git clone https://git.netfilter.org/libnftnl \
+  && cd libnftnl \
+  && ./autogen.sh \
+  && ./configure \
+  && make $make_args \
+  && make install && cp libnftnl.pc /usr/lib64/pkgconfig && make clean \
+  && cd / \
+  && rm -Rf libnftnl \
+  && git clone https://git.netfilter.org/iptables \
+  && cd iptables \
+  && ./autogen.sh \
+  && ./configure --prefix=/usr --sbindir=/sbin \
+  && make $make_args \
+  && make install && make clean \
+  && cd / \
+  && rm -Rf iptables
 RUN wget https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz \
```

Comment:
can you merge the two RUN commands using &&. That should reduce the size slightly

Reply:
Will do. thanks.

Reply:
done and amended the commit.